index.php

```diff
@@ -125,5 +84,72 @@ function isValidURL($url){
     
 <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<script type="text/javascript" charset="utf-8">
+;(function($) {
+  $(document).ready(function(){
+      var spinner = toggleSpinner();
+      $('form').submit(function(e){
+        e.preventDefault();
+        spinner();
+        $.ajax({
+          url: '/api/image-parser.php?url=' + $('input[name="url"]').val(),
+          // type : "POST",
+          dataType : 'json',
+          success : function(result) {
+            spinner();
+            if (result.success) {
+              renderStats(result);
+              renderImages(result);
+            } else {
+              $('#result').html('Invalid URL!');
+            }
+          },
+          error: function(xhr, resp, text) {
+            console.log('error', xhr, resp, text);
+          }
+        })
+      });
+  });
+
+  function renderStats(result) {
+    var stats = 
+      '<strong>URL Searched</strong> : <a href="' + result.url_searched + '" target="_blank">' + $('input[name="url"]').val() +'</a><br>' +
+      '<strong>Parent Domain</strong> : <a href="'+ result.parent_url +'" target="_blank">'+ result.parent_url+'</a><br>';
+    $('#stats .other-text').empty().append(stats);
+  }
+
+  function renderImages(result) {
+    var images;
+    if (result.images.length == 0) {
+      images = '<b>No Image Found at your Given Location</b>';
+    } else {
+      images = result.images.filter(onlyUnique).map(function(image) {
+        return '<a href="' + image +'"><img src="' + image + '" width="250" style="margin:20px"></a>';
+      });
+    }
+    $('#result').empty().append(images);
+  }
+
+  function toggleSpinner() {
+    var isHidden = true;
+    return function () {
+      if (isHidden) {
+        $('#stats').hide();
+        $('#result').empty();
+        $('.spinner').show();
+      }
+      else {
+        $('#stats').show();
+        $('.spinner').hide();
+      }
+      isHidden = !isHidden;
+    }
+  }
+
+  function onlyUnique(value, index, self) { 
```

Comment:
I don't think this is necessary, since API handles this
You can be sure that you are getting all the unique URLs